src/app/components/GameRun.tsx

```diff
@@ -1,31 +1,40 @@
-import React, { ReactNode, useState } from 'react'
-import { Game } from '../page'
-import Timer from './Timer'
-import TeamScorer from './TeamScorer'
-
-
+import React, { ReactNode, useState } from "react";
+import { Game } from "../page";
+import Timer from "./Timer";
+import TeamScorer from "./TeamScorer";
 
 type Props = {
-    gameSetup: Game;
-    setCurrentInterval: React.Dispatch<React.SetStateAction<number>>;
-    currentInterval: Number
-}
+  gameSetup: Game;
+  setCurrentInterval: React.Dispatch<React.SetStateAction<number>>;
+  currentInterval: number;
+};
 
-const GameRun = ({gameSetup, setCurrentInterval, currentInterval}: Props) => {
-    if(gameSetup){
```

Comment:
I moved the conditional rendering logic to the parent component. It shouldn't be the job of a component to work out what it should render when it is determined higher up in the tree (i.e. in page.tsx). So I changed this to conditionally render there instead, and GameRun can just be a "dumb" component whose sole job is to render data from its props.